README.md

```diff
@@ -83,7 +81,7 @@ yarn start
  
 - You can create an issue by clicking on this [link](https://github.com/AgileVentures/sfn-client/issues/new/choose) or by clicking on the new issue button on for [github issues](https://github.com/AgileVentures/sfn-client/issues) for the sfn-client project
 - Click on the [Get Started](https://github.com/AgileVentures/sfn-client/issues/new?assignees=&labels=&template=issue-template.md&title=) button to open the issue creation template. 
-- Fill in all the relevant sections as you create your issue.
+- Fill in all the relevant sections provided in the template as you create your issue.
 - Submit your issue by clicking on "Submit issue" button.
```

Comment:
This is an example of good opportunistic campground cleanup. It's not in the scope of the motivating issue, but it is in the path of the changes and I think it's a good add